keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -683,13 +696,29 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+If a control plane rollout disables this feature, the StatefulSet controller
+will update ordinal numbers it controls. This will result in pods being deleted,
+while other pods are scaled in. The StatefulSet controller scales up pods before
+it deletes pods, so as a result, the StatefulSet should not manage fewer than the
+number of replicas that are defined in the spec. Disabling the feature may have an effect on the
+stateful workload that is being run. If the stateful application expects a
+specific ordinal number to be available, it may result in an application failing
+to reach quorum, or rebalancing data based on the number of available replicas.
```

Comment:
Does an admin have a way of knowing that users are actively using this feature?

Reply:
For a specific StatefulSet, the `.spec.ordinals.start` field can be checked. If populated. I also added a metric `kube_statefulset_ordinals_start` and described in the "Monitoring Requirements" section. This can be used to count the number of StatefulSets within a cluster.